### PR TITLE
[Tests] Add microbenchmark to the new infra test

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

Verified it works. It also addresses the frequency comments from the previous PR

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: release/microbenchmark/wait_cluster.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import argparse
```
Comment (jjyao) [inline release/microbenchmark/wait_cluster.py:1]: why do we need this file?

File: release/release_tests.yaml [hunk 2]
```diff
@@ -768,6 +768,7 @@
 - name: shuffle_10gb
   group: Core nightly tests
   team: core
+  frequency: multi
```
Comment (jjyao) [inline release/release_tests.yaml:771]: multi not nightly? microbenchmark is nightly

File: release/microbenchmark/wait_cluster.py (hunk 1 above)
Comment (rkooo567) [reply release/microbenchmark/wait_cluster.py:1]: I have no idea why it is created... I feel like the run_release tests automatically create this. Let me remove it.

File: release/release_tests.yaml (hunk 2 above)
Comment (rkooo567) [reply release/release_tests.yaml:771]: This is for shuffle_10gb! (I realized this test wasn't running without specifying frequency)

**Review by jjyao — APPROVED** [review]
Assuming you remove wait_cluster

**Pull request merged**